auditable-serde/src/validation.rs

```diff
@@ -87,7 +87,7 @@ mod tests {
             source: Source::Local,
             kind: DependencyKind::Build,
             dependencies: deps,
-            root: root,
+            root,
```

Comment:
Is there a way to opt out of this globally? Like `#1[allow(this_lint)]` in main.rs? I'd prefer to keep these as is, rationale below.

I find explicit initialization more clear, even if it is more verbose.  This change takes advantage of a somewhat obscure and non-obvious feature of the language to save 6 characters. I'd rather type it out explicitly and make the code understandable even to people who don't know Rust very well.